sphinx/api/built-in/rose_bunch.rst

```diff
@@ -49,28 +49,33 @@ Configuration
 
 The application is normally configured in the
 :rose:conf:`rose_bunch[bunch]` and :rose:conf:`rose_bunch[bunch-args]`
-sections of the :rose:file:`rose-app.conf` file.
+sections of the :rose:file:`rose-app.conf` file, but `rose_bunch[command]`
+can be optionally used too, see below for details.
 
 .. rose:app:: rose_bunch
 
    .. rose:conf:: bunch
 
       .. rose:conf:: command-format=FORMAT
 
-         :compulsory: True
-
          A Pythonic ``printf``-style format string to construct the commands
          to run. Insert placeholders ``%(argname)s`` for substitution of the
          arguments specified under :rose:conf:`[bunch-args]` to the invoked
          command. The placeholder ``%(command-instances)s`` is reserved for
          inserting an automatically generated index for the command
-         invocation when using the command-instances setting.
+         invocation when using the command-instances setting. If not specified
+         then the command to run is determined following the ``mode=command``
+         logic, see :ref:`command-rose-app-run` for details, and arguments
+         are made accessible to the command as environment variables.
+
 
       .. rose:conf:: command-instances=N
 
          Allows the user to specify an integer value for the number of
          instances of a command they want to run. This generates the values
          used by the ``%(command-instances)s`` value in command-format.
+         If `[bunch]command-format` is not specified then the command instance
+         is passed as the ``COMMAND_INSTANCES`` environment variable instead.
```

Comment:
The `COMMAND_INSTANCES` environment variable can go in the environment variable section then be referenced as
```rst
:envvar:`COMMAND_INSTANCES`
```